python/ray/serve/tests/BUILD

```diff
@@ -30,9 +30,8 @@ py_test_module_list(
     "test_cluster_node_info_cache.py",
     "test_long_poll.py",
     "test_util.py",
-    "test_cross_language.py",
     "test_proxy.py",
-    "test_deployment_graph_ingress_deployment.py",
+    "test_deployment_graph_ingress.py",
```

Comment:
The name of the test is longer than windows limit (as we change bazel output from c/tmp to c/raytmp). Shorten it a bit.

Reply:
@zcin can we delete this test?

Reply:
feel free to let me know if you want to delete this one too!